source/dem/particle_wall_broad_search.cc

```diff
@@ -157,6 +157,10 @@ ParticleWallBroadSearch<dim>::particle_floating_mesh_contact_search(
       auto &candidates =
         particle_floating_mesh_contact_candidates[solid_counter];
 
+
+      // BB NOTE
+      // To look more into this
+
```

Comment:
Same here

Reply:
This one will remain, I'm still skeptical of the comment below.